alf/environments/gym_wrappers.py

```diff
@@ -229,11 +242,16 @@ def transform_space(self, observation_space):
             raise ValueError("Unsupported space:%s" % observation_space)
 
     def observation(self, observation):
-        for field in self._fields:
-            observation = transform_nest(
-                nested=observation,
-                field=field,
-                func=lambda obs: self.transform_observation(obs, field))
+        if self._fields is not None:
+            for field in self._fields:
+                observation = transform_nest(
+                    nested=observation,
+                    field=field,
+                    func=lambda obs: self.transform_observation(obs, field))
+        else:
```

Comment:
now this wrapper behaves differently from FrameStacker data transformer. FrameStacker data transformer does not have this "else" branch.

Reply:
> now this wrapper behaves differently from FrameStacker data transformer. FrameStacker data transformer does not have this "else" branch.

Reverted this change